styles/base.css

```diff
@@ -157,6 +185,35 @@ h4,
     line-height: 130%;
 }
 
+.h-sm,
+.blog_post h2 {
```

Comment:
The `h` refers to headline, and `sm` is the size of `sm`. 

The `.blog_post h2` has to be defined here like this, because we don't want blog writers to have to add CSS classes to Markdown.